libs/src/api/serviceProvider.js

```diff
@@ -40,12 +40,22 @@ class ServiceProvider extends Base {
     }
 
     // Time requests and get version info
-    const timings = await Utils.timeRequests(
-      creatorNodes.map(node => ({
-        id: node.endpoint,
-        url: `${node.endpoint}/version`
-      }))
-    )
+    let timings = (await Promise.all(
+      creatorNodes.map(async node => {
+        let resp
+        const request = {
+          id: node.endpoint,
+          url: `${node.endpoint}/version`
```

Comment:
per offline discussion, for consistency this should use the version validation code present in eth-contracts/index.js as well as this timing logic. the combination of both highest version and lowest latency should be what we end up selecting